alpha/veneer/basic/basic.go

```diff
@@ -15,30 +14,8 @@ type Veneer struct {
 	Registry image.Registry
 }
 
-func (v Veneer) Render(ctx context.Context, ref string) (*declcfg.DeclarativeConfig, error) {
-	// only taking first argument as file
-	stat, serr := os.Stat(ref)
-	if serr != nil {
-		return nil, serr
-	}
-
-	if stat.IsDir() {
-		return nil, errors.New("cannot render veneers by directory reference")
-	}
-	return v.renderFile(ctx, ref)
-}
-
-func (v Veneer) renderFile(ctx context.Context, ref string) (*declcfg.DeclarativeConfig, error) {
-	// xform any relative to absolute paths
-	abspath, err := filepath.Abs(ref)
-	if err != nil {
-		return nil, err
-	}
-	// xform to break apart dir/file elements
-	rpath, fname := filepath.Split(abspath)
-	root := os.DirFS(rpath)
-
-	cfg, err := declcfg.LoadFile(root, fname)
+func (v Veneer) Render(ctx context.Context, reader io.Reader) (*declcfg.DeclarativeConfig, error) {
+	cfg, err := declcfg.LoadReader(reader)
```

Comment:
We were originally using declcfg.LoadFile here because we wanted the follow-on call to readBundleObjects.  Do we no longer want that?

Reply:
`LoadFile` doesn't work with an arbitrary `io.Reader` because `readBundleObjects` needs to have some awareness of the directory in which the file containing the bundle object was found. That way it can de-reference those bundle object paths, using that directory as the root.